types/karma/index.d.ts

```diff
@@ -86,6 +86,7 @@ export class Server extends EventEmitter {
      * Force a refresh of the file list
      */
     refreshFiles(): Promise<any>;
+    refreshFile(path: string): Promise<any>;
```

Comment:
we can do better than a lazy `any` here. `refreshFiles` and `refreshFile` should both be a `Promise<void>`.

under the hood, both return various results (happy to explain if you really want) but just because shortcuts were used in the source to avoid wrapping promises (i.e. returning the result of another async method but not caring what that result is, for sake of avoiding wrapping it in a void promise). the intention iirc was that both are just void promises.

Reply:
I was not sure (not using those specific bits), trying to avoid scenario when list of files is returned in resolved promise.
I'll make a change though

Reply:
amended

Reply:
yeah these are really weird under the hood.

they're _actually_ something like:

```ts
refreshFiles(): Promise<void|{served: File[]}>;
refreshFile(path: string): Promise<void|{served: File[]}>;
```

but from what i remember they just did that to avoid having to wrap promises in void promises.

Reply:
ah did you push it? it still shows up as any :eyes: